packages/generator/templates/app/package.ts.json

```diff
@@ -27,7 +27,7 @@
     "@blitzjs/rpc": "latest",
     "@prisma/client": "3.9.0",
     "blitz": "alpha",
-    "next": "12.1.1",
+    "next": "latest",
```

Comment:
Are we sure about it? What if they push something that breaks Blitz? Shouldn't we pin it?